### DISCUSSION TITLE
Is it possible to change the date format in the File Browser (and/or in the git extension) ?

### DISCUSSION THREAD

**Discussion by diyoyo** [jupyterlab/jupyterlab#16116] [Q&A]
Hi, 
I would like to be able to read the real absolute date in the `Last Modified` column of the File Browser, instead of the relative date. Is that even possible?

Ideally, I would want the same in the git extension, but I may not be in the right forum to ask that question.

Thanks.

**Comment by jtpio** [comment]
Currently it's using the browser APIs to format the date:

https://github.com/jupyterlab/jupyterlab/blob/3d925e3a6cb2c5a3c1c4267406cad785e6fa2542/packages/coreutils/src/time.ts#L22-L47

Maybe there could be a user setting to allow changing the format of the dates?
